rollup/circuitcapacitychecker/impl.go

```diff
@@ -65,13 +67,14 @@ func (ccc *CircuitCapacityChecker) ApplyTransaction(traces *types.BlockTrace) (*
 		return nil, ErrUnknown
 	}
 
-	tracesByt, err := json.Marshal(traces)
+	ccc.jsonBuffer.Reset()
```

Comment:
What if we encounter an exceptionally large trace, would we just keep the allocated buffer, even though we almost never fully utilize it?

Reply:
Yeah, that is the trade-off here. Such peaks of memory usage become sticky.

Reply:
After disabling stack, I think we won't have such large traces as before.